src/flag_gems/ops/elu.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+import triton
+import triton.language as tl
+
+from ..utils import pointwise_dynamic
+
+
+@pointwise_dynamic(is_tensor=[True, False], promotion_methods=[(0, "DEFAULT")])
+@triton.jit
+def elu_forward_kernel(x, alpha):
+    return tl.where(x > 0, x, alpha * (tl.exp(x.to(tl.float32)) - 1))
+
+
+def elu(A, alpha=1.0):
```

Comment:
well, I think the last conversation is not resolved. you should define elu interface exactly the same as aten.